### PR TITLE
generate hasTag expressions

### PR DESCRIPTION
**Pull request by cdisselkoen**
Updates the generators for expressions to generate `.hasTag()` expressions.

### CODE REVIEW

File: cedar-policy-generators/src/expr.rs [hunk 1]
```diff
@@ -251,7 +251,24 @@ impl<'a> ExprGenerator<'a> {
...
                         ))
-                    })
+                    },
+                    7 => {
+                        let tag_name = uniform!(u,
+                            self.generate_expr(max_depth - 1, u)?,
+                            ast::Expr::val(self.schema.arbitrary_attr(u)?.0.clone())
+                        );
+                        let e = self.generate_expr(max_depth - 1, u)?;
+                        Ok(ast::Expr::get_tag(e, tag_name))
```
Comment (john-h-kastner-aws) [inline cedar-policy-generators/src/expr.rs:261]: Noting that you do generate `get_tag` also

**Review by john-h-kastner-aws — APPROVED** [review]
assuming you'll generate entity data with tags in a future PR

File: cedar-policy-generators/src/expr.rs (hunk 1 above)
Comment (cdisselkoen) [reply cedar-policy-generators/src/expr.rs:261]: Good catch.  We do not generate `get_tag` in all the places we would need to -- in particular we do not generate `get_tag` in the type-directed generators in this PR.  I will just remove this from this PR for clarity.

**Review by aaronjeline — APPROVED** [review]

**Comment by cdisselkoen** [comment]
CI failures are due to #448 not merged yet.  Merging this.

**Pull request merged**